Packages/Status/Sources/Status/Row/StatusRowView.swift

```diff
@@ -98,8 +98,12 @@ public struct StatusRowView: View {
       HStack(spacing: 2) {
         Image(systemName:"arrow.left.arrow.right.circle.fill")
         AvatarView(url: viewModel.status.account.avatar, size: .boost)
-        EmojiTextApp(viewModel.status.account.safeDisplayName.asMarkdown, emojis: viewModel.status.account.emojis)
-        Text("boosted")
+        if viewModel.status.account.username != account.account?.username {
+          EmojiTextApp(viewModel.status.account.safeDisplayName.asMarkdown, emojis: viewModel.status.account.emojis)
+          Text("boosted")
+        } else {
+          Text("You boosted")
+        }
```

Comment:
I couldn’t resist making this change. Feel free to revert. It _felt_ odd to me to see my name instead of it saying “you “.